concepts/05 Widgets/Sankey/99 Client-Side Exporting and Printing/01 User Interaction.md

```diff
@@ -86,4 +130,50 @@ Change the **export**.[formats](/api-reference/20%20Data%20Visualization%20Widge
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxSankey ... >
+            <DxExport
+                :enabled="true"
+                :formats="["PNG", "JPEG"]"
```

Comment:
let's move this array away from markup and declare it in the component <script> here and React.